### PR TITLE
DRV-522: always send query timeout header

### PR DESCRIPTION
**Pull request by vadimLF**
Added default query timeout to FaunaClient.

### CODE REVIEW

File: faunadb/client.go [hunk 1]
```diff
@@ -163,6 +163,9 @@ func NewFaunaClient(secret string, configs ...ClientConfig) *FaunaClient {
 
 	if client.queryTimeoutMs > 0 {
 		client.headers["X-Query-Timeout"] = strconv.FormatUint(client.queryTimeoutMs, 10)
+	} else {
+		client.queryTimeoutMs = uint64(requestTimeout.Milliseconds())
+		client.headers["X-Query-Timeout"] = strconv.FormatUint(uint64(requestTimeout.Milliseconds()), 10)
```
Comment (marrony) [inline faunadb/client.go:168]: That line is duplicated, it’s called in both branches of the if, why don’t you move it outside if and just update `client.queryTimeoutMs`

Comment (erickpintor) [reply faunadb/client.go:168]: I believe my comment on request vs query timeout also applies here: https://github.com/fauna/faunadb-jvm/pull/298#discussion_r583630192

Comment (vadimLF) [reply faunadb/client.go]: @erickpintor @marrony I updated PR, pls review.

**Review by parkhomenko — APPROVED** [review]

**Review by erickpintor — CHANGES_REQUESTED** [review]
This driver has a confusing pair of timeout configurations. `QueryTimeoutMS` sets now both query and request timeout, where `TimeoutMS` sets only the query timeout. I think we should merge them for simplicity.

**Comment by vadimLF** [comment]
> This driver has a confusing pair of timeout configurations. `QueryTimeoutMS` sets now both query and request timeout, where `TimeoutMS` sets only the query timeout. I think we should merge them for simplicity.

I removed `TimeoutMS` since we achieve the same functionality with `QueryTimeoutMS`

**Review by erickpintor — DISMISSED** [review]
LGTM

**Review by parkhomenko — DISMISSED** [review]

**Review by parkhomenko — APPROVED** [review]

**Review by erickpintor — APPROVED** [review]
re- 🐑 it.

**Pull request merged**